doc/cephfs/posix.rst

```diff
@@ -0,0 +1,37 @@
+========================
+ Differences from POSIX
+========================
+
+CephFS aims to adhere to POSIX semantics wherever possible.  For
+example, in contrast to many other common network file systems like
+NFS, CephFS maintains strong cache coherency across clients.  The goal
+is for processes communicating via the file system to behave the same
+when they are on different hosts as when they are on the same host.
+
+However, there are a few places where CephFS diverges from strict
+POSIX semantics for various reasons:
+
+- In shared simultaneous writer situations, a write that crosses
```

Comment:
Do we actually enable this case these days?  I thought that we only had two clients writing simultaneously with the old O_LAZY flag, otherwise we can only have one writing at a time.